apis/ec2/v1alpha1/custom_types.go

```diff
@@ -32,3 +32,54 @@ type CustomVPCPeeringConnectionParameters struct {
 	// connection if both VPCs are in the same tenant.
 	AcceptRequest bool `json:"acceptRequest,omitempty"`
 }
+
+// CustomTransitGatewayParameters are custom parameters for TransitGateway
+type CustomTransitGatewayParameters struct{}
+
+// CustomTransitGatewayVPCAttachmentParameters are custom parameters for TransitGatewayVPCAttachment
+type CustomTransitGatewayVPCAttachmentParameters struct {
+	// The ID of the VPC.
+	// +optional
```

Comment:
```suggestion
	// +optional
	// +crossplane:generate:reference:type=VPC
```

Can we use generated resolvers for these referenced fields?